apps/api/src/workspace/service/workspace.service.ts

```diff
@@ -763,7 +763,7 @@ export class WorkspaceService {
     //get all workspaces of user for page with limit
     const items = await this.prisma.workspace.findMany({
       skip: page * limit,
-      take: Math.min(limit, 30),
+      take: Number(limit),
```

Comment:
This won't be necessary. Just `limit` will do.

Reply:
Done 👍🏻

Reply:
You would need to pass ` limitMaxItemsPerPage(limit)` in here :(